src/components/HeaderNavigation/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styles from 'components/HeaderNavigation/Styles'
+import { getSubMenus } from 'components/navigationConfig'
+import { RnText, RnTouchableOpacity } from 'components/Rn'
+import React, { FC } from 'react'
+import { View } from 'react-native'
+import chatStore from 'stores/chatStore'
+
+const Navigation: FC<{
+  menu: string
+  subMenu: string
+}> = ({ menu, subMenu }) => {
+  return (
+    <View style={styles.navigation}>
+      {getSubMenus(menu).map(submenu => {
+        const { key, navFn, label } = submenu
+        const active = key === subMenu
+        const { unreadCount } = chatStore
+        const totalUnreadChat = unreadCount
```

Comment:
Destructured unreadCount can be used directly. No need to create another variable.